tests/e2e/bundle/parameters-reference-to-schemas/parameters.yaml

```
@@ -0,0 +1,5 @@
+Param:
+  in: query
```

invalid definition, missing name


```suggestion
  name: test
  in: query
```